Add fallbacks to all command translation strings

ChiefArug opened the pull request:
This solves https://github.com/neoforged/NeoForge/issues/1974 by ~~copying the translated text from the lang file into the source~~ loading the server's default translation for all neoforge command outputs, and ~~changing `translatable` methods to~~ using `translatableWithFallback`  to add those as fallbacks.

This also includes a couple of other changes that I noticed, namely two uses of `Component.translatable` being used for literals (changed to `Component.literal`) and one use of `translatableEscape` to `translatableWithFallback`, which shouldn't change anything as `translatableEscape` was being passed an empty array and all it does differently to `translatable` is modify the array.

ChiefArug commented:
Interesting, that caused the enumArgument test to fail as the 'text' as the test sees it is `Enum constant must be one of %1$s, found %2$s` (while it expects `Enum constant must be one of [BLD, NV], found ABC`), which points to a possible Mojang bug that `getString()` (what is used by the test) doesn't substitute in arguments.

sciwhiz12 reviewed (commented):
The duplication of the strings across both the language file and the source files concerns me a bit.

I seem to recall that the dedicated server loads the English (`en_us`) language data, so logged command output displays properly. Could we not make a helper method somewhere which gets the raw translated string from the language manager for a given key, and calls `translatableWithFallback` with those two?

That way, we avoid duplicating the translated strings, and we can use that method from now on in our commands and other places that may get sent to vanilla clients.

ChiefArug committed:
Start from scratch. Adds fallbacks to all command translation strings, pulled from the server's lang file.
This commit has not been tainted by Intellij's Optimise imports mucking everything up.

ChiefArug commented:
Done that, though I ran intellij's optimise imports which screwed over formatting (and couldn't be automatically fixed, gradle ugh) so just started from scratch as without the translated text in the code its a simple find and replace job. Ready for re-review.
